lib/Order.php

```diff
@@ -46,14 +46,13 @@ class Order extends ApiResource
      *
      * @throws \Stripe\Exception\ApiErrorException if the request fails
      *
-     * @return Order The paid order.
+     * @return OrderReturn The newly created return.
```

Comment:
This should be `\Stripe\OrderReturn` likely similar to collection from earlier, won't flag all